deploy/olm-catalog/kogito-operator/0.8.0/kogito-operator.v0.8.0.clusterserviceversion.yaml

```diff
@@ -133,6 +133,13 @@ spec:
         path: build.native
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      - description: EnableIstio annotates the pods managed by the operator with the
```

Comment:
same here.

Reply:
Generated code, will disappear once I remove from the source.